web/modules/custom/weather_block/src/Plugin/Block/WeatherBlock.php

```diff
@@ -70,57 +69,54 @@ public static function create(ContainerInterface $container, array $configuratio
   public function build(): array {
     $build = [];
     $user_id = $this->accountProxy->id();
-    $cityArray = $this->cityService->getSavedCityForUser($user_id);
-    if (empty($cityArray)) {
+    $citySaved = $this->cityService->getSavedCityForUser($user_id);
```

Comment:
`$city = $this->cityService->getSavedCityForUser($user_id) ?? $this->cityService->getDefaultCity()`